vyper/semantics/analysis/module.py

```diff
@@ -54,8 +54,8 @@
 def validate_module_semantics_r(
```

Comment:
why `_r`? seems like unexplained codebase definition to use these as external API, and `_r` should just be expanded

Reply:
`_r` is the convention for a lot of recursive functions. idk that it's really an external api, i can add a wrapper though